suite/tests/api/ir_aarch64_sve.c

```diff
@@ -8303,23 +8303,23 @@ TEST_INSTR(compact_sve)
 TEST_INSTR(str)
 {
     /* Testing STR <Zt>, [<Xn|SP>{, #<simm>, MUL VL}] */
-    int simm_0[6] = { 0, 255, -256, 127, -128, -1 };
+    int simm_0[6] = { 0, 32, -32, 8160, -4096, -8192 };
```

Comment:
Before it seemed to be checking boundary conditions: but the new numbers don't look like the same conditions?  Or at least they aren't a simple fixed scale-up.  Ditto below.  How did you come up with the new numbers?

Reply:
For test vector length of 256 bits the `simm_0` array does include boundary values:
255 x 32 bytes -> 8160
-256 x 32 bytes -> -8192

Updated two test values for `STR <Pt>, [<Xn|SP>{, #<simm>, MUL VL}]` and `LDR <Zt>, [<Xn|SP>{, #<simm>, MUL VL}]` which weren't boundary values.

Reply:
OK, I was just expecting the same scale factor in the diff: so if -256 became -32, -128 should become -16, but it became -4096 instead.